src/app/components/Timestamp/index.jsx

```diff
@@ -22,34 +22,15 @@ const StyledTimestamp = styled.span`
   }
 `;
 
-const Timestamp = ({ children, datetime, prefix, suffix }) => {
-  if (prefix || suffix) {
-    return (
-      <StyledTimestamp>
-        {prefix}
-        <time dateTime={datetime}>{children}</time>
-        {suffix}
-      </StyledTimestamp>
-    );
-  }
-
-  return (
-    <StyledTimestamp as="time" dateTime={datetime}>
-      {children}
-    </StyledTimestamp>
-  );
-};
+const Timestamp = ({ children, datetime }) => (
+  <StyledTimestamp as="time" dateTime={datetime}>
```

Comment:
Does this need to be `as="time"`, cant it be a styled `time` element originally?